pkg/reconciler/emit_metrics.go

```diff
@@ -12,7 +12,7 @@ func (r *Reconciler) emitMetrics(pr *tektonv1.PipelineRun) error {
 	eventType := pr.GetAnnotations()[keys.EventType]
 
 	switch gitProvider {
-	case "github":
```

Comment:
@savitaashture a question about this (should have been ask in the original pr that goes to main) would that make the github-enteprise metrics reported as github ?

I am guessing we want to different those ?

same goes for gitlab gitea bitbucket-cloud bitbucket-server, should not be differentiated ?

Reply:
If I understand your question correctly, are you asking whether when metrics for GitLab are reported, it shows 'GitLab' or not?

If so, yes, the reported metrics will include the provider from which they originated, like below

For GitLab

![Screenshot from 2024-03-22 13-53-23](https://github.com/openshift-pipelines/pipelines-as-code/assets/9441662/d04d7fdb-1c6e-4d33-a1fb-206544e8284f)

For GitHubEnterprise

![Screenshot from 2024-03-22 14-01-52](https://github.com/openshift-pipelines/pipelines-as-code/assets/9441662/acaf3dbf-f809-459d-9c95-a7afad9bfb3a)